### ISSUE TITLE
Pattern library failing (storybook) - cannot find Jcrop.gif

### ISSUE THREAD

**Issue by lb-** [wagtail/wagtail#8646]
### Issue Summary

Since the merging of stylesheets  (https://github.com/wagtail/wagtail/pull/8564) the pattern library (storybook) build is failing locally.

### Steps to Reproduce

- run the pattern library locally as per  https://docs.wagtail.org/en/latest/contributing/developing.html#using-the-pattern-library

### Details
* commenting out the line...
in `wagtail/client/scss/vendor/jquery.Jcrop.min.css` fixes it.

```
.jcrop-vline,.jcrop-hline{background:#FFF url(../../wagtailimages/images/Jcrop.gif);font-size:0;position:absolute;}

```

```
resolve '../../wagtailimages/images/Jcrop.gif' in '/Users/lb/dev/wagtail/wagtail-dev/wagtail/wagtail/admin/static_src/wagtailadmin/scss'
  using description file: .../wagtail/wagtail-dev/wagtail/package.json (relative path: ./wagtail/admin/static_src/wagtailadmin/scss)
    using description file: .../wagtail/wagtail-dev/wagtail/package.json (relative path: ./wagtail/admin/static_src/wagtailimages/images/Jcrop.gif)
      no extension
.../wagtail/wagtail-dev/wagtail/wagtail/admin/static_src/wagtailimages/images/Jcrop.gif doesn't exist
      .mjs
.../wagtail/wagtail-dev/wagtail/wagtail/admin/static_src/wagtailimages/images/Jcrop.gif.mjs doesn't exist
      .js
.../wagtail/wagtail-dev/wagtail/wagtail/admin/static_src/wagtailimages/images/Jcrop.gif.js doesn't exist
      .jsx
.../wagtail/wagtail-dev/wagtail/wagtail/admin/static_src/wagtailimages/images/Jcrop.gif.jsx doesn't exist
      .ts
.../wagtail/wagtail-dev/wagtail/wagtail/admin/static_src/wagtailimages/images/Jcrop.gif.ts doesn't exist
      .tsx
.../wagtail/wagtail-dev/wagtail/wagtail/admin/static_src/wagtailimages/images/Jcrop.gif.tsx doesn't exist
      .json
.../wagtail/wagtail-dev/wagtail/wagtail/admin/static_src/wagtailimages/images/Jcrop.gif.json doesn't exist
      .cjs
        .../wagtail/wagtail-dev/wagtail/wagtail/admin/static_src/wagtailimages/images/Jcrop.gif.cjs doesn't exist
      as directory
        /...wagtail/wagtail-dev/wagtail/wagtail/admin/static_src/wagtailimages/images/Jcrop.gif doesn't exist
```

_Originally posted by @lb- in https://github.com/wagtail/wagtail/pull/8564#discussion_r891944503_

**Comment by lb-** [comment]
@PaarthAgarwal  / @thibaudcolas  here is the issue.

I am not sure on the solution but will try to have a look tomorrow if possible.

**Comment by lb-** [comment]
Discovered in https://github.com/wagtail/wagtail/pull/8638#issuecomment-1149550202

**Comment by lb-** [comment]
Additionally, once we fix this, we may want to add some kind of pattern library check to our CI (build the static output and dump it or something similar).

**Comment by thibaudcolas** [comment]
Fixed in #8651, could you take a look @lb-?
